### PR TITLE
adding support for the check and formatWithCursor api calls

### PR DESCRIPTION
**Pull request by h223chen**
Honestly not too sure what the best approach for this should be, but given that it's a grunt task i figured it should just take a param. It should default to 'format' if are no params. I can write unit tests for this but wanted to put it up first to check the overall approach

### CODE REVIEW

File: tasks/prettier.js [hunk 1]
```diff
@@ -101,29 +116,43 @@ function prettierTask(grunt) {
...
+
           if (progress) {
             bar.tick();
           } else {
             grunt.log.writeln('Prettify file "' + filepath + '".');
           }
         });
+
+        if (!checkStatus) {
+          grunt.fail.warn('some files not pretty')
```
Comment (poalrom) [inline tasks/prettier.js]: Think first word should starts with a capital letter

**Review by poalrom — COMMENTED** [review]
Nice work! Lets write unit tests and merge this to master

**Comment by h223chen** [comment]
@poalrom unit test have been added

**Comment by poalrom** [comment]
Nice. Thanks for pr

**Pull request merged**